src/Behat/Behat/Gherkin/ServiceContainer/GherkinExtension.php

```diff
@@ -97,6 +99,11 @@ public function configure(ArrayNodeDefinition $builder)
                     ->useAttributeAsKey('name')
                     ->prototype('scalar')->end()
                 ->end()
+                ->enumNode('parser')
+                    ->info('Sets which gherkin parser to use if not specified at CLI')
+                    ->values(['auto', 'cucumber', 'legacy'])
```

Comment:
Legacy -> Behat?

Reply:
Yeah I don't like either name :)

Legacy is too harsh but 'behat' is confusing as the new one is used by behat too. Any suggestions?

Reply:
Yeah, `behat` is not my favourite too, but makes it consistent with `cucumber`? What about if we make it a package name? `behat/gherkin` or `cucumber/gherkin` - could be more discoverable this way.

Reply:
How about `classic`?